apps/omg_watcher/lib/omg_watcher/release_tasks/set_exit_processor_sla_margin.ex

```diff
@@ -18,8 +18,8 @@ defmodule OMG.Watcher.ReleaseTasks.SetExitProcessorSLAMargin do
   require Logger
   @app :omg_watcher
 
-  @system_env_name_margin "EXIT_PROCESSOR_SLA_MARGIN"
-  @app_env_name_margin :exit_processor_sla_margin
+  @system_env_name_margin "EXIT_PROCESSOR_SLA_SECONDS"
```

Comment:
Note that before merging this PR, we'll need to update our infra to have this environment variable so we don't break the CI/CD environment.